tests/test_snakemake.py

```diff
@@ -86,9 +112,9 @@ def test_nucleotide_search_containment(request, genome_file, database_name):
 @pytest.mark.dependency()
 @pytest.mark.parametrize("genome_file", test_genomes)
 @pytest.mark.parametrize("database_name", prot_databases)
-def test_translate_search_containment(request, genome_file, database_name):
-    depends(request, [f"test_translate_sketch[{g}]" for g in test_genomes])
-    target = f"search-containment/{genome_file}.x.{database_name}.search-contain-matches.sig"
+def test_translate_contigs_taxonomy(request, genome_file, database_name):
+    depends(request, [f"test_translate_search_containment[{genome_file}, {database_name}]"])
```

Comment:
@ ctb do you know how to pass in multiple files?

Reply:
yes! pass them in as a tuple. can give you example code upon request (in an hour), but there's something analogous [here](https://docs.pytest.org/en/stable/parametrize.html#parametrize-basics) --

```
# content of test_expectation.py
import pytest


@pytest.mark.parametrize(
    "test_input,expected",
    [("3+5", 8), ("2+4", 6), pytest.param("6*9", 42, marks=pytest.mark.xfail)],
)
def test_eval(test_input, expected):
    assert eval(test_input) == expected
```

Reply:
thx! Need to check if I'm thinking about this correctly -- The current `parameterization` is working  (doing all combinations of `genome_file` and `database_name`) up through the `search_containment` tests. But the `contigs_taxonomy` tests are getting skipped bc the `depends` code isn't passing in those params properly.

If I change the parameterization to a single tuple (with both values), would that enable properly passing that in through `depends`?